db/migrate/20161125161211_add_all_day_to_holidays.rb

```diff
@@ -0,0 +1,10 @@
+class AddAllDayToHolidays < ActiveRecord::Migration[5.0]
+  def up
+    add_column :holidays, :all_day, :boolean, default: false, null: false
+    execute 'UPDATE holidays SET all_day = true WHERE bank_holiday = true'
```

Comment:
Fair enough, I usually do this sort of thing so that we don't have to rely on class names in case they change in the future.